frontend/express/app.js

```diff
@@ -654,11 +654,27 @@ app.use(function(err, req, res, next) { // eslint-disable-line no-unused-vars
 
 
 //prevent bruteforce attacks
-bruteforce.db = countlyDb;
-bruteforce.mail = countlyMail;
-bruteforce.paths.push(countlyConfig.path + "/login");
-bruteforce.paths.push(countlyConfig.path + "/mobile/login");
-app.use(bruteforce.defaultPrevent);
+rateLimiter.db = countlyDb;
+rateLimiter.mail = countlyMail;
+
+for (let path of ["/login", "/mobile/login"]) {
+    const absPath = countlyConfig.path + path;
+    rateLimiter.collections[absPath] = "failed_logins";
+    rateLimiter.identifiers[absPath] = function(req) {
+        return {username: req.body.username}
+    };
+    rateLimiter.blockHooks[absPath] = function(id, req, res) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Functions declared within loops referencing an outer scoped variable may lead to confusing semantics. (rateLimiter)](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=5182840)